INSTALL.md

```diff
@@ -0,0 +1,62 @@
+# Setup
+
+This repository supports:
+
+- [Linux](#unix)
+- [macOS](#unix)
+- [Windows Subsystem for Linux 2](#wsl-2)
+- [Docker](#docker) (any host OS)
+
+If you encounter any issues setting up the repo, please feel free to [reach out to us on Discord](https://discord.gg/urUm3VG).
+
+
+## Unix
+
+Clone the repository:
+```sh
+git clone https://github.com/pmret/papermario
+cd papermario
+```
+
+Install build dependencies:
+```sh
+./install.sh
+```
+
+Copy a Paper Mario (USA) ROM into the root directory of the repository with the name `baserom.z64`. If you're using WSL, you can enter the Linux filesystem by opening `\\wsl$` in File Explorer; e.g. `\\wsl$\Ubuntu\home\<your username>\papermario`.
```

Comment:
I think we should add the sha1 hash here.

Reply:
`./configure.py` actually checks that its valid now, so I think it would be redundant, Additionally the README now links to the database (hashes, size, etc.) for the ROM we expect

Reply:
at least make mention of it, imo

Reply:
should be resolved now